context.go

```diff
@@ -409,11 +409,7 @@ func (c *context) JSON(code int, i interface{}) (err error) {
 	if c.echo.Debug || pretty {
 		return c.JSONPretty(code, i, "  ")
 	}
-	b, err := json.Marshal(i)
-	if err != nil {
-		return
-	}
-	return c.JSONBlob(code, b)
+	return c.jsonBlob(code, i)
```

Comment:
Since `jsonBlob` is simple, the code can be moved directly here.

Reply:
yeah! nice job